pkg/volume/local/local_test.go

```diff
@@ -197,8 +222,63 @@ func TestMountUnmount(t *testing.T) {
 	}
 }
 
+func TestMapUnmap(t *testing.T) {
```

Comment:
We should mention that this method is testing block.

Reply:
Fixed.